app/bot.py

```diff
@@ -0,0 +1,269 @@
+	2:"What is the location where this incident took place?",
+	3:"Thanks! You're helping (align incentives)!",
+	4:"Thanks anyway!",
+	5:"Please fill out this form "
+}
+
+# test_entries = [
+# 	{"root_tweet_id":"1424511565932359685","tweeter_id":"witt_rowen","conversation_status":0,"tweet_text":"Incident report test 1 🦟📡","checks_made":0},
+# 	{"root_tweet_id":"1424420702208237573","tweeter_id":"witt_rowen","conversation_status":0,"tweet_text":"Incident report test 2 🐥","checks_made":0},
+# 	{"root_tweet_id":"1424452159744077826","tweeter_id":"witt_rowen","conversation_status":0,"tweet_text":"Incident report test 3🦧","checks_made":0},
+# ]
+
+test_entries = [
+	{"form":1,"incident_id":1,"isChecked":False,"link":"https://a.humanrightsfirst.dev/edit/1426290795267731463","tweet_id":"1424511565932359685","user_name":"witt_rowen"}
+]
+
+test_insert = [
+	{"city":"TestTown","confidence":None,"dsecription":"testtest","force_rank":"Rank - 1 Police Presence","incident_date":"2021-04-22T00:00:00.000Z","incident_id":1,"lat":42.64249,"long":-73.7576,"src":[],"tags":["police","bacon","general_pork_products"],"title":"testers","tweet_id":"1424511565932359685","user_name":"witt_rowen"}
+]
+
+
+def send_form(data:Dict):
+	""" Sends form to user, inserts to conversations table """
+
+	to_insert = DB.convert_invocation_conversations(data)
+	to_insert['root_tweet_id'] = int(to_insert['root_tweet_id'])
+	to_insert['tweet_text'] = '@' + to_insert['user_name'] + ' ' + conversation_tree[5] + to_insert['link'] 
+	to_insert['reachout_template'] = conversation_tree[5]
+	del to_insert['link']
+	del to_insert['user_name']
+	try:
+		status = twitter.respond_to_tweet(to_insert['root_tweet_id'], to_insert['tweet_text'])
+		to_insert['tweeter_id'] = bot_name
+		to_insert['isChecked'] = True
+		to_insert['conversation_status'] = 6
+		to_insert['checks_made'] = 1
+		to_insert['sent_tweet_id'] = status.id,
+
+		DB.insert_data_conversations([to_insert])
+	except tweepy.TweepError as e:
+		logging.error("Tweepy error occured:{}".format(e))
+
+
+def receive_form(data:Dict):
+	""" Takes input from user/POST inputs into conversations if no root_id with conversation_status 7 """
+
+	to_insert = DB.convert_form_conversations(data)
+	to_insert['root_tweet_id'] = int(to_insert['root_tweet_id'])
+	to_insert['isChecked'] = True
+	to_insert['conversation_status'] = 7
+	validation_check = DB.get_root_seven(to_insert['root_tweet_id'])
+	if len(validation_check) == 0:
+		DB.insert_data_conversations([to_insert])
+
+
+def advance_all():
+	""" Advances all conversations based on highest conversations status per root_tweet_id """
+	to_advance = DB.get_to_advance()
+	for threads in to_advance:
+		advance_conversation(points.root_tweet_id)
+
+
+def reset_conversations_for_test():   # DELETE BEFORE PR
+	DB.reset_conversations()   # DELETE BEFORE PR
+	DB.insert_data_conversations(test_entries)   # DELETE BEFORE PR
+	print("done, you idiot")   # DELETE BEFORE PR
+
```

Comment:
multiple delete before PR comments not deleted

Reply:
Removed comments, kept function as testing is crucial and this function is useful for doing testing between two twitter accounts -- other option is to send tweets to real users